static/js/lib/video_test.js

```diff
@@ -79,4 +80,14 @@ describe('video library functions', () => {
       });
     });
   });
+
+  describe('uploadToDropbox', () => {
+    it('calls the Dropbox API with correct arguments', () => {
+      const sandbox = sinon.sandbox.create();
+      window.Dropbox = {save: () => {}};
+      const dropboxStub = sandbox.stub(window.Dropbox, 'save');
+      saveToDropbox(video);
+      sinon.assert.calledWith(dropboxStub, `undefined${video.key}/video.mp4`);
```

Comment:
i think we can/should set the settings value and check it here instead of asserting that it's `undefined` (ex: https://github.com/mitodl/odl-video-service/blob/master/static/js/containers/ErrorPage_test.js#L66)

Reply:
OK